### PR TITLE
Make all aliases have periods + Add hyphenated taxis to alias ref doc

### LINKED ISSUE #69: fix .tr .trhs .trcr .trf aliases so they don't handle multi-taxiway routes wonky
currently, .tr 8R N N14 M yields  "Runway 8R, taxi via N. N14 M" due to the period at the end. Others need evaluation.

### PR DESCRIPTION
**Pull request by ksnifte**
Closes #69, per @FriedrichKayak's request

### CODE REVIEW

**Comment by FriedrichKayak** [comment]
Was your manual testing successful to prove the inclusion of the period didn't break anything?

**Comment by ksnifte** [comment]
@FriedrichKayak Yes, testing of all changed aliases was successful

File: docs/alias/Alias.txt [hunk 1]
```diff
@@ -213,7 +213,7 @@ TOWER
 .GA GO AROUND.
 .MISS fly the missed approach as published.
 .CG contact ground.
-.CGF contact ground, $1
+.CGF contact ground, $1.
```
[the remark this answers is not in the document]
Comment (ksnifte) [reply docs/alias/Alias.txt:216]: @erikquinn I don't disagree, although I asked @FriedrichKayak and he said to include the period.  Personally, I think the period is worth it, but that is neither here nor there-- what do the two of you guys want?

(same applies to the rest of the below comments, obviously)

File: docs/alias/Alias.txt [hunk 2]
```diff
@@ -34,7 +34,7 @@ GENERAL
 .UOF are you on frequency?
 .BRB ATTENTION ALL AIRCRAFT: $callsign will be away for approximately $1 minute(s).
 .BACK $callsign has returned.
-.PRC For explanations/questions/tips, please visit the VATSIM pilot resource center at www.vatsim.net/prc/
+.PRC For explanations/questions/tips, please visit the VATSIM pilot resource center at www.vatsim.net/prc/.
```
[the remark this answers is not in the document]
Comment (ksnifte) [reply docs/alias/Alias.txt:37]: Good point, will do

File: docs/alias/Alias.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (erikquinn) [reply docs/alias/Alias.txt:216]: I have a feeling that was a generalization that he answered to, and not specifically regarding aliases that end in a frequency. I'm fine with it if that's what he wants to do-- we'll await clarification on it for now.

**Pull request merged**